packages/ui/core-components/src/lib/unsorted/viz/map/BaseMap.svelte

```diff
@@ -44,6 +45,15 @@
 
 	const allGeoJsonLoaded = evidenceMap.allGeoJsonLoaded;
 
+	let legendData = evidenceMap.legendData;
+
+	//Question: Do i need to specify, if i default legend position to 'bottomLeft' in other areas it gets used?
+	export let legendPosition = 'bottomLeft';
```

Comment:
You need to specify it at least anywhere that is user facing, but it doesn't hurt to specify it everywhere